vizro-core/examples/_dev/app.py

```diff
@@ -30,7 +34,62 @@
     ],
 )
 
-dashboard = vm.Dashboard(pages=[page])
+# CREATE FAKE DATA
```

Comment:
Changes will be reverted. Added the page for testing purpose.

Reply:
It's fine to leave it in here, no need to worry about this becoming a mess - it's what it's there for 🙂